src/clj/rems/api/user_settings.clj

```diff
@@ -0,0 +1,31 @@
+(ns rems.api.user-settings
+  (:require [compojure.api.sweet :refer :all]
+            [rems.api.schema :refer :all]
+            [rems.db.user-settings :as user-settings]
+            [rems.util :refer [getx-user-id]]
+            [ring.util.http-response :refer :all]
+            [schema.core :as s]))
+
+(s/defschema UpdateUserSettingsCommand
+  {:user-id s/Str
+   (s/optional-key :language) s/Keyword})
+
+(s/defschema UserSettings
+  {(s/optional-key :language) s/Keyword})
+
+(def user-settings-api
+  (context "/user-settings" []
+    :tags ["user-settings"]
+
+    (GET "/" []
+      :summary "Get user settings"
+      :roles #{:logged-in}
+      :return UserSettings
+      (ok (user-settings/get-user-settings (getx-user-id))))
+
+    (PUT "/update" []
```

Comment:
This too could be "/" for a more REST style.

Reply:
Done